examples/src/main/java/org/eclipse/rdf4j/examples/model/Example03LiteralDatatypes.java

```diff
@@ -60,8 +53,7 @@ public static void main(String[] args) {
 					int peopleDepicted = literal.intValue();
 					System.out.println(peopleDepicted + " people are depicted in this painting");
 				} else if (property.getLocalName().equals("creationDate")) {
-					XMLGregorianCalendar calendar = literal.calendarValue();
-					Date date = calendar.toGregorianCalendar().getTime();
+					LocalDate date = LocalDate.from(literal.temporalAccessorValue());
```

Comment:
That doesn't seem very intuitive. Do we not have any `.toLocalDate()` and `.toLocalDateTime()` methods?

Reply:
We don't - at least not yet. Agree that that might be a good addition. This is already easier than the old mucking about with an XMLGregorianCalendar object though.